manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
The default class should be chained after the package.
The package should be installed first, then the default class will replace the stock defaults file in that order.